tests/unit/swagger/definitionsGeneration/definitions.spec.ts

```diff
@@ -333,6 +353,15 @@ describe('Definition generation', () => {
                 genericNestedArrayCharacter2: (propertyName, propertySchema) => {
                     expect(propertySchema.$ref).to.eq('#/definitions/GenericRequestTypeAliasModel2Array', `for property ${propertyName}.$ref`);
                 },
+                and: (propertyName, propertySchema) => {
+                  expect(propertySchema.type).to.eq('object');
```

Comment:
All of these definition tests should be verifying that `additionalProperties` is set to the expected value. That goes for schemaDetails3.spec.ts too.

Reply:
https://github.com/lukeautry/tsoa/pull/400/commits/62bac808fb811cab908398bf24e637a484c29ce5